pkg/util/util.go

```diff
@@ -229,6 +229,8 @@ func IsOwnedByTidbCluster(obj metav1.Object) (bool, *metav1.OwnerReference) {
 
 // RemainNodeport is to make service nodeport unchanged during each reconciliation
 func RemainNodeport(desiredSvc, existedSvc *corev1.Service) {
```

Comment:
I think the function Name should be changed. What about `RemainExistServiceConfiguration` ?